src/viking.js

```diff
@@ -1,11 +1,116 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength;
+  }
+
+  receiveDamage(damage) {
+    this.health = this.health - damage;
```

Comment:
```suggestion
    this.health -= damage;
```
Os lo pongo asi porque normalmente nunca lo vereis como lo habeis escrito, asi que cuanto antes os acostumbreis pues genial!